### PR TITLE
Add support for dhl-paket

### PR DESCRIPTION
**Pull request by fkryvyts-ingrid**

### CODE REVIEW

File: drawer.go [hunk 1]
```diff
@@ -44,12 +50,30 @@ func (d *Drawer) DrawLabelAsPng(label elements.LabelInfo, output io.Writer, opti
...
-			err := drawer.Draw(gCtx, element, options)
+			err := drawer.Draw(gCtx2, element, options)
 			if err != nil {
 				return fmt.Errorf("failed to draw zpl element: %w", err)
 			}
 		}
+
+		if reversePrint {
+			xorImg := images.NewXor(gCtx2.Image(), gCtx.Image())
+			gCtx.DrawImage(xorImg, 0, 0)
```
Comment (mknycha-ingrid) [inline drawer.go:75]: You're doing XOR and the output is like that text, right?
Screenshot 2024-03-21 at 16 42 46
Shouldn't it be called "inversion" instead or "reverse"?

Comment (fkryvyts-ingrid) [reply drawer.go]: Yeah, renamed it to ReversePrint

File: drawers/text_field.go [hunk 2]
```diff
@@ -62,11 +67,25 @@ func NewTextFieldDrawer() *ElementDrawer {
...
+func getTopLeftPos(gCtx *gg.Context, bLX, bLY float64, text *elements.TextField) (float64, float64) {
+	var h float64
+
+	if text.Block != nil {
+		_, h = gCtx.MeasureMultilineString(text.Text, float64(text.Block.LineSpacing))
+	} else {
+		_, h = gCtx.MeasureString(text.Text)
+	}
+
+	h *= 0.7
```
[the remark this answers is not in the document]
Comment (fkryvyts-ingrid) [reply drawers/text_field.go:79]: well, it is indeed magic
I have no idea why I need to multiple it by this value to get the same results as labelary but without this positions are a bit off

File: drawer.go (hunk 1 above)
Comment (fkryvyts-ingrid) [reply drawer.go]: As for why it is called Reverse and not Inverse, it is named like this in ZPL manuals/specifications
Screenshot from 2024-03-21 18-26-57

File: drawers/text_field.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (fkryvyts-ingrid) [reply drawers/text_field.go]: Replaced with fontOffset variable

[the remark this answers is not in the document]
Comment (fkryvyts-ingrid) [reply drawers/text_field.go]: Rewrote logic completely, that magic variable is no longer needed

File: testdata/text_fo_n.zpl [hunk 3]
```diff
@@ -0,0 +1,4 @@
+^XA^FO30,30^A0N,,28^FR^FDGOGREEN^FS^XZ
+
```
Comment (mknycha-ingrid) [inline testdata/text_fo_n.zpl:2]: Nitpick: Unnecessary empty lines

**Review by mknycha-ingrid — APPROVED** [review]

File: testdata/text_fo_n.zpl (hunk 3 above)
Comment (fkryvyts-ingrid) [reply testdata/text_fo_n.zpl]: good catch, removed

**Pull request merged**